### ISSUE TITLE
Report E2BIG spawn E2BIG

### ISSUE THREAD

**Issue by cabarique** [vknabel/vscode-swiftlint#21]
```
Error: spawn E2BIG
	at ChildProcess.spawn (internal/child_process.js:394:11)
	at spawn (child_process.js:553:9)
	at Object.execFile (child_process.js:237:17)
	at Object.module.<computed> [as execFile] (electron/js2c/asar.js:163:31)
	at /Users/pro/.vscode/extensions/vknabel.vscode-swiftlint-1.2.4/out/lint.js:154:38
	at new Promise (<anonymous>)
	at execSwiftlint (/Users/pro/.vscode/extensions/vknabel.vscode-swiftlint-1.2.4/out/lint.js:153:12)
	at Object.<anonymous> (/Users/pro/.vscode/extensions/vknabel.vscode-swiftlint-1.2.4/out/lint.js:59:38)
	at Generator.next (<anonymous>)
	at fulfilled (/Users/pro/.vscode/extensions/vknabel.vscode-swiftlint-1.2.4/out/lint.js:5:58)
	at processTicksAndRejections (internal/process/task_queues.js:85:5)
os: darwin
```
_URL decoded contents by @vknabel_

**Comment by vknabel** [comment]
@cabarique Thanks for the report!

Do you have any special setup?
Is the project where this error occurred open source, so I can reproduce the issue under the same circumstances?
Can you provide the path (or the length of it) to your repo?
Do you have any extension specific config? What does your swiftlint config look like?
Does it only happen in specific files? If yes: which one?

If you wonder regarding the weird questions: This might be related to long file paths and large projects.

**Comment by vknabel** [comment]
Closed in favor of #22. Hoping there will be more information
